proxy/coco/src/state.rs

```diff
@@ -1149,6 +1149,7 @@ pub mod test {
         );
 
         assert!(!project_names.contains(&fakie.subject().name.to_string()));
+        drop(tmp_dir); // ensure repo lives until end of test
```

Comment:
Could you explain why you added this? Without the `drop()` Rust should keep `tmp_dir` around until it goes out of scope, i.e. here. Maybe you were running into this error (#1378)?

```
test can_ask_and_clone_project ... fatal: '.' does not appear to be a git repository
```

Reply:
In this case we can remove it. I had a few tests on previously where the tmpdir was moved into a method, and was dropped before finishing the test.

You are right, the method should drop by the end of it. I've added the `drop` as a guardrail to prevent moving the tmpdir and dropping it earlier with a compiler error.

Reply:
Not relevant anymore. The file is gone.